docs/build-walkthrough/build-process.md

```diff
@@ -3,24 +3,32 @@ title: Development Guide
 sidebar_label: Development Guide
 ---
 
-To immediately start building on the UMA protocol, first check out the [Quick Start guide](/developers/setup) for setting up your environment and deploying your first local [synthetic token](/build-walkthrough/mint-locally).
+The UMA development guide aims to assist developers during the build process through tutorials, technical explanations, and frequently used resources.
```

Comment:
Welcome to the UMA Development guide. This page provides links for product build tutorials, technical explanations and frequently used resources.